### PR TITLE
Update to preview 5

### PR DESCRIPTION
**Pull request by Gillibald**

### CODE REVIEW

File: src/AvaloniaEdit/Rendering/VisualLineText.cs [hunk 1]
```diff
@@ -68,14 +68,9 @@ public override TextRun CreateTextRun(int startVisualColumn, ITextRunConstructio
 				offset,
 				DocumentLength - relativeOffset);
 
-            var bufferOffset = RelativeTextOffset;
+            //var bufferOffset = RelativeTextOffset;
```
Comment (danipen) [inline src/AvaloniaEdit/Rendering/VisualLineText.cs:71]: Can we remove this commented code?

**Review by danipen — APPROVED** [review]
LGTM

**Comment by danipen** [comment]
@Gillibald after the upgrade, the button text, and the status bar text is black instead of white. Are we using the dark theme for the application?
image

**Comment by Takoooooo** [comment]
> @Gillibald after the upgrade, the button text, and the status bar text is black instead of white. Are we using the dark theme for the application? image

I guess we should use ThemeVarian Light in Demo app

**Comment by Takoooooo** [comment]
Ping me when this PR will be merged i will make a release

**Comment by danipen** [comment]
@Takoooooo it is already merged. You can make the release.

**Pull request merged**